tests/k8s/bootstrap-w-conf.yml

```diff
@@ -1,39 +0,0 @@
-apiVersion: extensions/v1beta1
```

Comment:
I thought we're leaving the option for using only bootstrap. (for hare/p2p/sync)..

Reply:
You are right. I'll leave this file here and handle its use for p2p/hare in another PR